pkgs/by-name/ne/netop/package.nix

```
@@ -0,0 +1,27 @@
+{ lib, libpcap, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "netop";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "ZingerLittleBee";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Rnp2VNAi8BNbKqkGFoYUb4C5db5BS1P1cqpWlroTmdQ=";
+  };
+
+  LIBPCAP_LIBDIR = lib.makeLibraryPath [ libpcap ];
+  LIBPCAP_VER = libpcap.version;
+
+  cargoSha256 = "5vbv4w17DdaTKuF3vQOfv74I8hp2Zpsp40ZlF08qWlc=";
+
+  meta = with lib; {
+    description = "Network Top -- Help you monitor network traffic with bpf";
```

Maybe something like this for the short description. I suggest the change because the package name should not be included here (:
```suggestion
    description = "Network monitor network traffic using bpf";
```

---

Thanks for your review. I've addressed your other points, but I don't understand this one. Network top is not the name of the program, netop is. :) This description was taken from the upstream github repo.

---

Thanks, and I see your points on the name. I still think that the description would better without the 'Network Top' as don't directly describes what the package does and is more succinct expressed by stating that it is just a network monitor using bpf.

However, I made a mistake in my suggestion as well. So here is a new one that makes a bit more sense.
```suggestion
    description = "A network monitor using bpf";
```

[Cite:](https://github.com/NixOS/nixpkgs/blob/3cb442f49442e3dc76ecc661fbe236362396da8e/doc/stdenv/meta.chapter.md?plain=1)
### `description` {#var-meta-description}

A short (one-line) description of the package.
This is displayed on [search.nixos.org](https://search.nixos.org/packages).

Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package --- just describe what it does.

Wrong: `"libpng is a library that allows you to decode PNG images."`

Right: `"A library for decoding PNG images"`